Assets/PatchKit Patcher/Editor/PatcherManifestCreator.cs

```diff
@@ -8,7 +8,33 @@ namespace PatchKit.Unity.Editor
 {
     public static class PatcherManifestCreator
     {
-        private const int ManifestVersion = 3;
+        private const int ManifestVersion = 4;
+
+        [MenuItem("Tools/PatchKit Patcher/Manifest/Windows")]
+        private static void Test(Manifest manifest)
```

Comment:
Are you sure we want to leave test methods here?

Reply:
These methods serve as a quick way to manually verify if the manifest generates correctly.

I'll change their names though to make it clearer.